addons/binding/org.openhab.binding.seneye/src/main/java/org/openhab/binding/seneye/handler/SeneyeHandler.java

```diff
@@ -83,17 +83,17 @@ public void invalidConfig() {
     public void initialize() {
         SeneyeConfigurationParameters config = getConfigAs(SeneyeConfigurationParameters.class);
 
-        if (config.aquarium_name.isEmpty()) {
+        if (config.aquarium_name != null && config.aquarium_name.isEmpty()) {

```

Comment:
I think you cannot combine both cases here. if the `aquarium_name` is `null` you also want to set the thing to `OFFLINE`, right?

Reply:
I guess it can, The user made an item file and the property aquarium_name is not present (or is miss spelled, since there is no default value, it will be null in the config class.
It can be empty if it is not filled in (in the UI or in the item file).
I guess both cases should say the setting is missing and put the thing offline, there is a manual intervention needed to make things right.

Reply:
Yes but in your code if `config.aquarium_name` is `null` you are skipping this branch and do **not** set it to `OFFLINE` or am I thinking in the wrong direction?

That is the only point I have left in my reviews so far.